apps/src/templates/RegionalPartnerSearch.jsx

```diff
@@ -562,7 +592,12 @@ const styles = {
   bigButton: {
     padding: '10px 20px 10px 20px',
     height: 'initial',
-    marginTop: 22,
+    //marginTop: 22,
```

Comment:
Nit: could this line be deleted or are you keeping it in case the top margin is desired?

Reply:
almost certainly I commented this to see if it fixed something and forgot to actually delete it 😅